tilp/trunk/src/tilp_config.c

```diff
@@ -102,90 +102,68 @@ int tilp_config_default(void)
 	return 0;
 }
 
+static void deprecated_config_path(char* old_ini_file, char* ini_file)
+{
+	int old_exists, new_exists;
+	old_exists = !access(old_ini_file, F_OK);
+	new_exists = !access(ini_file, F_OK);
+
+	if (old_exists && !new_exists)
+	{
+		FILE *in;
+		FILE *out;
+
+		in = fopen(old_ini_file, "rb");
+		out = fopen(ini_file, "wb");
+		if (in && out)
+		{
+			int c;
+			while ((c = fgetc(in)) != EOF)
+			{
+				fputc(c, out);
+			}
+			fclose(out);
+			fclose(in);
+			// A config file now exists at the new location.
+			// Delete the old file.
+			unlink(old_ini_file);
+		}
+	}
+}
+
 static char * get_config_path(void)
 {
-	return g_strconcat(g_get_home_dir(), G_DIR_SEPARATOR_S, INI_FILE, NULL);
+	return g_build_path(G_DIR_SEPARATOR_S, g_get_user_config_dir(), "tilp.ini", NULL);
 }
 
+#if defined(__LINUX__) || defined(__BSD__) || defined(__MACOSX__)
+# define OLD_INI_FILE  "tilp"
```

Comment:
@adriweb it seems the leading dot was accidentally removed here when it was moved from `tilp_paths.h`
thanks for noticing that!